deploy/cap/source.rb

```diff
@@ -1,6 +1,9 @@
 # frozen_string_literal: true
 
 require "pathname"
+require "rsync"
+
+RSYNC_OPTIONS = ["-v", "-r", "-l", "-p"].freeze
```

Comment:
I don't think we want verbose unless activated by DEBUG (or fauxpaas -v).

I don't think we want symlinks, so we have a self-contained artifact.

I think we do want `z` for compression.

Reply:
* Verbosity is controller by the runner. There's no output if a quiet runner is used. We get the desired behavior without testing for flags here.
* We don't want symlinks that go outside the artifact, for sure. I feel like we want internal ones. With this flag, we don't modify the artifact in-flight.
* added  compression